### PR TITLE
Refactoring code for readability

### PR DESCRIPTION
**Pull request by chelseawinfree**

### CODE REVIEW

File: symantecssl/request_models.py [hunk 1]
```diff
@@ -92,12 +92,12 @@ def serialize(self, order_type):
...
-                root, 'ProductCode', self.product_code
-            )
-            utils.create_subelement_with_text(
-                root, 'PartnerOrderID', self.partner_order_id
-            )
+            for node, node_text in [
+                ('ProductCode', self.product_code),
+                ('PartnerOrderID', self.partner_order_id)
+            ]:
+                utils.create_subelement_with_text(root, node, node_text)
```
Comment (jmvrbanac) [inline symantecssl/request_models.py]: Suggestion for a slightly different way of doing this.

```python
element_data = {
    'ProductCode': self.product_code,
    'PartnerOrderID': self.partner_order_id
}

[utils.create_subelement_with_text(root, node, node_text)
 for node, node_text in element_data.items()]
```

Comment (reaperhulk) [inline symantecssl/request_models.py]: Don't use a dict unless these fields aren't order dependent :)

Comment (jmvrbanac) [inline symantecssl/request_models.py]: @chelseawinfree are the fields order dependent?

Comment (jmvrbanac) [inline symantecssl/request_models.py]: I'm fine leaving with leaving as is, but something to investigate in the future. :+1:

Comment (chelseawinfree) [inline symantecssl/request_models.py]: Not sure exactly, i would have to test every section. Symantec's code is very ... specific in certain things. This order matches their docs though, (for all instances) and I would uch prefer to leave it this way.

Comment (reaperhulk) [inline symantecssl/request_models.py]: :+1:

Comment (jfwood) [inline symantecssl/request_models.py]: :+1:

**Comment by jfwood** [comment]
:100:

**Pull request merged**